atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -446,7 +458,7 @@ public synchronized void shutdown() {
         if (daemon == null) {
             return;
         }
-        log.debug("Signalling background sweeper to shut down.");
+        log.info("Signalling background sweeper to shut down.");
```

Comment:
This will be logged at WARN `log.warn("Shutting down background sweeper. Please restart the service to rerun background sweep.");`.. this will just be spammy.